deliver/lib/deliver/options.rb

```diff
@@ -99,7 +99,7 @@ def self.available_options
                                      optional: true,
                                      default_value: "ios",
                                      verify_block: proc do |value|
-                                       UI.user_error!("The platform can only be ios, appletvos, or osx") unless %('ios', 'appletvos', 'osx').include?(value)
+                                       UI.user_error!("The platform can only be ios, appletvos, xros or osx") unless %('ios', 'appletvos', 'xros', 'osx').include?(value)
```

Comment:
Should the supported platform name here be `xros` or `visionos`?

I’m thinking it should be the latter, as xrOS is just a leftover from the beta period before Apple decided on / communicated the visionOS name, so it’s effectively superseded by `visionOS` (even though it’s of course still used internally by Xcode SDKs).

It would definitely feel more natural to me to write `platform :visionos do ...` than to use `:xros`, but so this is not a question specific to Deliver but a general Fastlane question.